CONTRIBUTING.md

```diff
@@ -70,7 +70,7 @@ There are many ways to contribute,
 from writing new exercises and improving existing ones
 to updating or filling in the documentation
 and submitting [bug reports][issues]
-about things that don't work, aren't clear, or are missing.
+about things that do not work, are not clear, or are missing.
```

Comment:
these changes should be propagated up to `styles`. I'll open an issue there.

Reply:
Or not...where does the stock CONTRIBUTING.md file live?

Reply:
it is in `styles`, but in the `boilerplate` directory used to create these files when the lesson is initialised: https://github.com/carpentries/styles/blob/gh-pages/bin/boilerplate/CONTRIBUTING.md